### ISSUE TITLE
mpt removes package files on a failed update without prompt

### ISSUE THREAD

**Issue by payonel** [magik6k/MPT2#11]
If a user is updating a mpt package -- and that package fails to update due to a file conflict (e.g. the mpt package has a new file that already exists on the system such as /bin/rm.lua) then the install fails but also removes the other existing files of that package.

expected: no update or removal of existing files occur. (optionally, a feature is added to mpt to also allow the user to force the update/install - to overwrite the existing file conflicts).
actual: all existing files of the mpt package are removed from the system.

**Comment by magik6k** [comment]
Yeah, I'll probably add a prompt asking if conflicts should be ignored(i.e. whether to skip the files or replace them), not nice solution but still better
